stacks/dhis2-core/helmfile.yaml

```diff
@@ -27,7 +27,7 @@ releases:
       - dhisConfig: |
           connection.dialect = org.hibernate.dialect.PostgreSQLDialect
           connection.driver_class = org.postgresql.Driver
-          connection.url = jdbc:postgresql://{{ requiredEnv "DATABASE_HOSTNAME" }}/{{ env "DATABASE_DATABASE" | default "dhis2" }}
+          connection.url = jdbc:postgresql://{{ requiredEnv "INSTANCE_NAME" }}-database-postgresql/{{ env "DATABASE_DATABASE" | default "dhis2" }}
```

Comment:
should this be changed? if so in the dhis2-core-deploy script DATABASE_HOSTNAME is still provided as required parameter